services/121-service/src/payments/payments.service.ts

```diff
@@ -319,10 +317,26 @@ export class PaymentsService {
     return bulkActionResultPaymentDto;
   }
 
-  // ##TODO needs to be refactored to work with programFspConfigProperties
-  async validateRequiredFinancialServiceProviderConfigurations(
+  private async checkFspConfigurationsOrThrow(
```

Comment:
I tested this functionally by dropping columnToMatch for gringotts and get a generic error in the front-end, correct error in back-end. Is that good enough?

![image](https://github.com/user-attachments/assets/1222b740-1ead-4d3d-845c-1bba87df8f33)